### DISCUSSION TITLE
WEB3，我們要進到下一階段了嗎

### DISCUSSION THREAD

**Discussion by Tomlord1122** [1101-GS4719-C/Python-Community-GS4719-C#221] [Show and tell]
其實有一個概念，叫做WEB3，我們的網路世代進入到了第三階段，以block chain、加密貨幣、元宇宙為主，其實我個人認為現在有很多亂七八糟的項目，且也沒有一個法規保護，現在有很多詐騙網站，仿冒opensea，來詐騙虛擬貨幣，但這項技術是我們應該要持續關注的，也有很多有趣的項目大家可以去研究，像是defi、鏈遊等等。

淺談一下我發現到的虛擬貨幣目前的問題，大家有想法也可以在下面留言，虛擬貨幣主打的是擁有一個完整的交易序列，我們可以知道每個貨幣的交易紀錄等等且過程中都會加密來處理，那這個錢除非是帳號被盜走，不然很難被別人給偷走，保障了很多，但以台灣相關的法規來看，如果這個虛擬貨幣的主人死掉的話，他的這些錢就會永遠的消失了嗎，畢竟大家都沒有他的帳號，財產的分配一定非常的困難，我覺得這應該是一個問題

**Comment by Isongzhe** [comment]
感覺還不到下個階段的程度，像之前 薩爾瓦多正式將比特幣作為國家法定貨幣與美元並列使用。就造成非常大的問題。

**Comment by Joan0518** [comment]
台灣對於這類的法規還很多沒有跟進，不過最近設立了數位發展部，表面上看起來要做事了，實際不知道如何
